helm/vitess/templates/_orchestrator.tpl

```diff
@@ -60,7 +55,7 @@ kind: StatefulSet
 metadata:
   name: orchestrator
 spec:
-  serviceName: orchestrator-headless
+  serviceName: orchestrator
```

Comment:
I think this needs to stay `orchestrator-headless` since (I believe) per-Pod DNS only works with headless services.